src/components/20-molecules/datepicker/index.scss

```diff
@@ -30,9 +29,10 @@ axa-datepicker {
     }
   }
 
-  &[error]:not([error=""]),
+  &[error]:not([error='']),
```

Comment:
Why change to single quotes? I don’t know if this works in all browsers. Standard is “”. I would revert

Reply:
Prettier is fixing that...?! Will check my prettier rules.

Reply:
IntelliJ is not using .prettierignore if i start prettier manuelly. Will change back to `"`

Reply:
I agree with Luca, make it double quotes.